CIS-3760-Team-25/unity-checkers
# In-Game Interface

**Description**

Adds the in-game overlay displaying current player and captures. Also includes some minor UI improvements


**Testing**

Manual Testing
 - Tested that the in-game overlay is displayed when the game starts
 - Tested that the overlay is updated when a turn ends
 - Tested that the overlay is updated when a piece is captured
 - Tested that the overlay home button displays a warning
    - Tested that the warning can end the game and return to the main menu
    - Tested that the warning can be closed and the game can be resumed
 - Tested that the overlay is reset when the game is restarted from game over screen
 - Tested that the overlay is reset when the game is started from player select
    

**Related Issues**

Closes #39


**Definition of Done**

Before Merging

- [x] Code conforms to coding standards (formatted/linted)
- [x] Code is properly documented (comments, model updated)
- [x] Code has been tested and the testing method is documented in this PR
- [x] Code doesn't introduce any new bugs
- [x] Code is reviewed by at least 2 people
- [x] Kanban board is updated

After Merging

- [ ] GitHub branches related to the story are deleted
- [ ] Associated GitHub issue is updated/closed
- [ ] Kanban board is updated
